document-parsers/langchain4j-document-parser-llamaparse/src/test/java/dev/langchain4j/data/document/parser/llamaparse/LlamaParseClientIT.java

```diff
@@ -0,0 +1,69 @@
+package dev.langchain4j.data.document.parser.llamaparse;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.Duration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.with;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@EnabledIfEnvironmentVariable(named = "LLAMA_PARSE_API_KEY", matches = ".+")
+public class LlamaParseClientIT {
+
+    private static String API_KEY;
+    private static LlamaParseClient client;
+    private static String JOB_ID;
+
+    @BeforeAll
+    static void prepareTest() {
+        API_KEY = System.getenv("LLAMA_PARSE_API_KEY");
+        client = LlamaParseClient.builder()
+                .apiKey(API_KEY)
+                .baseUrl(LlmaParseApi.baseUrl)
+                .timeout(Duration.ofSeconds(5))
+                .build();
+    }
+
+    @Test
+    @Order(1)
+    void shouldParseFile() {
+        Path path = Paths.get(this.getClass().getResource("/files/sample.pdf").getPath());
```

Comment:
This fails on my machine with:

`java.nio.file.InvalidPathException: Illegal char <:> at index 2: /C:/dev/repo/langchain4j_1/document-parsers/langchain4j-document-parser-llamaparse/target/test-classes/files/sample.pdf`

You can use this to read the file from resources:

```java
private Path toPath(String fileName) {
        try {
            return Paths.get(getClass().getClassLoader().getResource(fileName).toURI());
        } catch (URISyntaxException e) {
            throw new RuntimeException(e);
        }
}

Path path = toPath("files/sample.pdf");
```

Reply:
done